pom.xml

```diff
@@ -26,7 +26,7 @@
     <maven.compiler.release>11</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus.version>3.2.6.Final</quarkus.version>
+    <quarkus.version>3.5.0.CR1</quarkus.version>
```

Comment:
I am not sure I'd like to upgrade it right now, as it would make it incompatible with the 3.2 LTS version

Reply:
change is now backward compatible. I left version `3.2.6`